scripts/artifacts-building/apt/aptly-vars.yml

```diff
@@ -42,33 +45,52 @@ aptly_architectures:
-    - "slushie-{{ rpc_release }}-elastic-kibana-4.5-ALL"
-    - "slushie-{{ rpc_release }}-elastic-beats-ALL"
-    - "slushie-{{ rpc_release }}-elastic-es-1.7-ALL"
-    - "slushie-{{ rpc_release }}-elastic-es-2.x-ALL"
-  xenial:
-    - "slushie-{{ rpc_release }}-ubuntu-updates-xenial"
-    - "slushie-{{ rpc_release }}-uca-newton-updates-xenial"
-    - "slushie-{{ rpc_release }}-mariadb-10.0-xenial"
-    - "slushie-{{ rpc_release }}-percona-xenial"
-    - "slushie-{{ rpc_release }}-rabbitmq-downloaded-packages-{{ rpc_release }}-ALL"
-    - "slushie-{{ rpc_release }}-elastic-logstash-2.3-ALL"
-    - "slushie-{{ rpc_release }}-elastic-kibana-4.5-ALL"
-    - "slushie-{{ rpc_release }}-elastic-beats-ALL"
-    - "slushie-{{ rpc_release }}-elastic-es-1.7-ALL"
-    - "slushie-{{ rpc_release }}-elastic-es-2.x-ALL"
+  r14:
+    trusty:
+      - "slushie-{{ rpc_release }}-ubuntu-updates-trusty"
+      - "slushie-{{ rpc_release }}-ubuntu-backports-trusty"
+      - "slushie-{{ rpc_release }}-uca-mitaka-updates-trusty"
+      - "slushie-{{ rpc_release }}-mariadb-10.0-trusty"
+      - "slushie-{{ rpc_release }}-percona-trusty"
+      - "slushie-{{ rpc_release }}-haproxy-trusty"
+      - "slushie-{{ rpc_release }}-rabbitmq-downloaded-packages-{{ rpc_release }}-ALL"
+      - "slushie-{{ rpc_release }}-elastic-logstash-2.3-ALL"
+      - "slushie-{{ rpc_release }}-elastic-kibana-4.5-ALL"
+      - "slushie-{{ rpc_release }}-elastic-beats-ALL"
+      - "slushie-{{ rpc_release }}-elastic-es-1.7-ALL"
+      - "slushie-{{ rpc_release }}-elastic-es-2.x-ALL"
+    xenial:
+      - "slushie-{{ rpc_release }}-ubuntu-updates-xenial"
+      - "slushie-{{ rpc_release }}-uca-newton-updates-xenial"
+      - "slushie-{{ rpc_release }}-mariadb-10.0-xenial"
+      - "slushie-{{ rpc_release }}-percona-xenial"
+      - "slushie-{{ rpc_release }}-rabbitmq-downloaded-packages-{{ rpc_release }}-ALL"
+      - "slushie-{{ rpc_release }}-elastic-logstash-2.3-ALL"
+      - "slushie-{{ rpc_release }}-elastic-kibana-4.5-ALL"
+      - "slushie-{{ rpc_release }}-elastic-beats-ALL"
+      - "slushie-{{ rpc_release }}-elastic-es-1.7-ALL"
+      - "slushie-{{ rpc_release }}-elastic-es-2.x-ALL"
+  r15:
+    xenial:
+      - "slushie-{{ rpc_release }}-ubuntu-updates-xenial"
+      - "slushie-{{ rpc_release }}-uca-ocata-updates-xenial"
+      - "slushie-{{ rpc_release }}-mariadb-10.1-xenial"
+      - "slushie-{{ rpc_release }}-percona-xenial"
+      - "slushie-{{ rpc_release }}-rabbitmq-downloaded-packages-{{ rpc_release }}-ALL"
+      - "slushie-{{ rpc_release }}-elastic-logstash-2.3-ALL"
+      - "slushie-{{ rpc_release }}-elastic-kibana-4.5-ALL"
+      - "slushie-{{ rpc_release }}-elastic-beats-ALL"
+      - "slushie-{{ rpc_release }}-elastic-es-1.7-ALL"
+      - "slushie-{{ rpc_release }}-elastic-es-2.x-ALL"
+
 # mapping for N (NOT MERGE) snapshots
 # This is a list of the repo/mirror snapshots (slushies) that will be published separately.
 aptly_n_mapping:
```

Comment:
What do you mean? They will be tagged as:
- slushie-r15.0.0rc1-hwraid-trusty
- slushie-r15.0.0rc1-hwraid-xenial

But we can just ignore them in the meantime